### PR TITLE
hello-http: Add version 1.4.1

### PR DESCRIPTION
**Pull request by Slach**
A cross-platform HTTP client desktop application for testing HTTP and REST APIs, WebSocket, GraphQL (including subscriptions) and gRPC endpoints.

### CODE REVIEW

**Comment by Slach** [comment]
@aliesbelik could you run /verify again?

File: bucket/hello-http.json
Comment (aliesbelik) [inline bucket/hello-http.json:5]:
```suggestion
```

This appearance should be removed (since considered as link to the files to download).

**Comment by aliesbelik** [comment]
> @aliesbelik could you run /verify again?

BTW (and for other PRs you will raise in future), running check is the step required from PR submitter (as per [contribution guideline](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md)):

> Add a comment starting with "/verify" after raising the PR - this will kick in the automatic manifest verifier.

**Review by aliesbelik — CHANGES_REQUESTED** [review]

File: bucket/hello-http.json
Comment (Slach) [reply bucket/hello-http.json:5]: but why JSON schema still contains this field as required?
I added url because IDEA suggest me, it makred  are required
https://github.com/ScoopInstaller/Scoop/blob/master/schema.json#L649-L651

**Comment by Slach** [comment]
@aliesbelik could you suggest? why verifier think hashes is wrong?

**Review by aliesbelik — APPROVED** [review]
Looks good to me.

**Comment by Slach** [comment]
@aliesbelik could you explain what need to PR be merged? maybe share link with merge process description?

**Comment by aliesbelik** [comment]
> @aliesbelik could you explain what need to PR be merged? maybe share link with merge process description?

As from my experience PRs into `extras` are [not very quickly processed](https://github.com/ScoopInstaller/Extras/discussions/9291).
Wait for maintainers review (I'm not one).

**Pull request merged**
